packages/components/src/entities/SampleAliquotsGridPanel.tsx

```diff
@@ -195,17 +218,13 @@ interface SampleAliquotsGridPanelProps extends Props {
 }
 
 export const SampleAliquotsGridPanel: FC<SampleAliquotsGridPanelProps> = props => {
-    const { sampleId, sampleLsid, schemaQuery, rootLsid, user, omittedColumns } = props;
-    const id = createGridModelId(
-        'sample-aliquots-' + sampleId,
-        SchemaQuery.create(SCHEMAS.SAMPLE_SETS.SCHEMA, schemaQuery.getQuery())
-    );
+    const { sampleLsid, schemaQuery, rootLsid, user, omittedColumns } = props;
     const omitted = omittedColumns
         ? [...getOmittedSampleTypeColumns(user), ...omittedColumns]
         : getOmittedSampleTypeColumns(user);
 
     const queryConfigs = {
-        [id]: getSampleAliquotsQueryConfig(schemaQuery.getQuery(), sampleLsid, true, rootLsid, omitted),
+        [MODEL_ID]: getSampleAliquotsQueryConfig(schemaQuery.getQuery(), sampleLsid, true, rootLsid, omitted),
```

Comment:
nit: This could just be `model` instead of `[MODEL_ID]` and then the underlying component can contractually understand to look at `queryModels.model`. The `MODEL_ID` is superfluous.